File: 006/006.c
#include <stdio.h>

int main() {
    int sumSquare = 0, squareSum = 0;
    for (int i = 0; i <= 100; i++) {
        squareSum += i * i;
        sumSquare += i;
    }
    printf("%d\n", sumSquare * sumSquare - squareSum);
}

File: 006/006.rs
fn main() {
    let mut sum_square = 0;
    let mut square_sum = 0;
    for i in 1..100 {
        square_sum += i * i;
        sum_square += i;
    }
    println!("{}", sum_square * sum_square - square_sum);
}
